### PR TITLE
feat: added option to throw on missing env

### PR DESCRIPTION
**Pull request by fivethreeo**
Based on https://github.com/browserslist/browserslist/pull/560

* Added `throwOnMissing` option to browserslist
* Added tests for `throwOnMissing` option to browserslist
* Added jsdoc for option but found no script for updating .d.ts
* Coverage not full but can't figure out why

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -82,6 +82,14 @@ function pickEnv(config, opts) {
     name = 'production'
   }
 
+  if (opts.throwOnMissing) {
```
Comment (ai) [inline node.js:85]: Where do we set default value?

**Comment by ai** [comment]
Thanks. Released in 4.19.

**Comment by fivethreeo** [comment]
Is the d.ts built from jsdoc? Seems to be missing the option in the typing.

**Comment by ai** [comment]
Nope. We have separated `.d.ts` file. Can you fix it in PR?

**Pull request merged**